packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1140,6 +1140,10 @@ class CupertinoDialogAction extends StatelessWidget {
     TextStyle style = _kCupertinoDialogActionStyle;
     style = style.merge(textStyle);
 
+    if (isDefaultAction) {
```

Comment:
It appears that `isDefaultAction` can be explicitly set to `null` in the constructor. Could you add an assert to `CupertinoDialogAction`'s constructor to make sure `isDefaultAction` is not nullable?